### PR TITLE
[IOS-495] PreviousLookBrick

### PR DESCRIPTION
**Pull request by thePsguy**

### CODE REVIEW

File: src/Catty.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -277,6 +277,14 @@
 		4CFAD35F1CFAB3EC0019C636 /* UIImageExtensionTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 4CFAD35D1CFAB3EC0019C636 /* UIImageExtensionTests.swift */; };
 		4CFAD3611CFAB9AA0019C636 /* transparency-rgba.png in Resources */ = {isa = PBXBuildFile; fileRef = 4CFAD3601CFAB9AA0019C636 /* transparency-rgba.png */; };
 		4CFC3BF71CF1972500DF5AA3 /* CatrobatReorderableCollectionViewFlowLayout.m in Sources */ = {isa = PBXBuildFile; fileRef = 4CFC3BF61CF1972500DF5AA3 /* CatrobatReorderableCollectionViewFlowLayout.m */; };
+		4D6E0C8F1EE07213008FED9F /* IfThenLogicBeginBrick.m in Sources */ = {isa = PBXBuildFile; fileRef = 4D6E0C8E1EE07213008FED9F /* IfThenLogicBeginBrick.m */; };
```
Comment (m-herold) [inline src/Catty.xcodeproj/project.pbxproj]: Something went wrong with your project file. The IfThenLogicBeginBrick(+CBXMLHandler) files should not be part of this PR.

File: src/Catty.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1253,6 +1261,20 @@
 		4CFAD3601CFAB9AA0019C636 /* transparency-rgba.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; name = "transparency-rgba.png"; path = "CattyTests/Resources/Images/transparency-rgba.png"; sourceTree = SOURCE_ROOT; };
 		4CFC3BF51CF1972500DF5AA3 /* CatrobatReorderableCollectionViewFlowLayout.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = CatrobatReorderableCollectionViewFlowLayout.h; sourceTree = "<group>"; };
 		4CFC3BF61CF1972500DF5AA3 /* CatrobatReorderableCollectionViewFlowLayout.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = CatrobatReorderableCollectionViewFlowLayout.m; sourceTree = "<group>"; };
+		4D6E0C8B1EE07201008FED9F /* IfThenLogicBeginBrick.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = IfThenLogicBeginBrick.h; sourceTree = "<group>"; };
```
Comment (m-herold) [inline src/Catty.xcodeproj/project.pbxproj]: see comment above.

File: src/Catty/DataModel/Bricks/Look/NextLookBrick+CBXMLHandler.h [hunk 3]
```diff
@@ -0,0 +1,28 @@
+/**
```
Comment (m-herold) [inline src/Catty/DataModel/Bricks/Look/NextLookBrick+CBXMLHandler.h]: This file has the wrong name and is in the wrong directory.

File: src/Catty/DataModel/Bricks/Look/NextLookBrick+CBXMLHandler.m [hunk 4]
```diff
@@ -0,0 +1,46 @@
+/**
```
Comment (m-herold) [inline src/Catty/DataModel/Bricks/Look/NextLookBrick+CBXMLHandler.m]: This file has the wrong name and is in the wrong directory.

Comment (thePsguy) [inline src/Catty/DataModel/Bricks/Look/NextLookBrick+CBXMLHandler.m]: In XCode, this file and the above file show up in the correct directory. Looks like when I duplicated and modified the CBXMLHandler for the `PreviousLookBrick`, I forgot to rename it even though I updated it's contents, and the XCode for some reason physically moved it to another directory while keeping it logically in the same directory as intended. I'm fixing this right now.
